src/main/java/programminglife/gui/controller/GuiController.java

```diff
@@ -357,7 +356,7 @@ private void initLeftControlpanelDraw() {
                 FXMLLoader loader = new FXMLLoader(ProgrammingLife.class.getResource("/CreateBookmarkWindow.fxml"));
                 AnchorPane page = loader.load();
                 GuiCreateBookmarkController gc = loader.getController();
-                gc.setGraphController(graphController);
+                gc.setGuiController(this);
```

Comment:
The bookmarks window should also be updated with the new bookmark when this window is opened.